src/app/static/src/app/types.ts

```diff
@@ -153,4 +153,9 @@ export interface UploadFile {
     resourceId: string | null,
     resourceUrl: string | null,
     lastModified: string | null
+}
+
+export interface BarchartConfig {
```

Comment:
I think you should be able to import the `AxisConfig` type from `@reside-ic/vue-charts' instead of having to declare it yourself.